rdflib/namespace.py

```diff
@@ -95,11 +95,6 @@ def __new__(cls, value):
             rt = unicode.__new__(cls, value, 'utf-8')
         return rt
 
-
-    @property
-    def title(self):
-        return URIRef(self + 'title')
-
```

Comment:
Note: the title method won't be called (of course), but the value will be `<built-in method upper of Namespace object at ...>` instead of (e.g.) dcterms:title.

(This is a general gotcha with the Namespace utility colliding with unicode method names of course, and I guess it would be more ideal if it used `__getattribute__` to never return bound methods.)